applications/RomApplication/python_scripts/rom_analysis.py

```diff
@@ -19,9 +19,16 @@ def __init__(self,global_model, parameters):
 
         def _CreateSolver(self):
             """ Create the Solver (and create and import the ModelPart if it is not alread in the model) """
+            self.rom_basis_output_name = 'RomParameters' #Default
+            if self.project_parameters.Has("output_processes"):
+                for name, _ in self.project_parameters["output_processes"].items():
+                    if name=="rom_output":
+                        rom_output_paramaters = self.project_parameters["output_processes"]["rom_output"]
+                        rom_basis_output_format = rom_output_paramaters[0]["Parameters"]["rom_basis_output_format"].GetString()
+                        self.rom_basis_output_name = rom_output_paramaters[0]["Parameters"]["rom_basis_output_name"].GetString()
```

Comment:
This is a bit hacky. I'm starting to think that now that we have unified ROM solver and analysis plus the rom manager, maybe we can directly call the rom_output_process in the analysis stage rather than including it in the output processes.

We should at least consider and discuss this option.

Reply:
Agree. We'll need to expose some settings. In any case, lets do what you suggest, we merge this and then we discuss with the rest of the team how we approach this issue.